python/doc/examples/calibration/least_squares_and_gaussian_calibration/plot_generate_chaboche.py

```diff
@@ -0,0 +1,188 @@
+
+Parameters to calibrate
+-----------------------
+
+The vector of parameters to calibrate is:
+
+.. math::
+
+   \\theta = (R,C,\\gamma).
+
+
+The true values of the parameters are:
+
+- :math:`R = 750\\times 10^6`,
+- :math:`C = 2750\\times 10^6`,
+- :math:`\\gamma = 10`.
+
+This is the set of *true* values that we wish to estimate with the calibration
+methods.
+In practical studies, these values are unknown.
+
+Observations
+------------
+
+In order to create a calibration problem, we make the hypothesis that the
+strain has the following distribution:
+
+.. math::
+   \\epsilon \\sim \\mathcal{U}(0,0.07).
+
+where :math:`\\mathcal{U}` is the uniform distribution.
+Moreover, we consider a Gaussian noise on the observed constraint:
+
+.. math::
+   \\epsilon_\\sigma \\sim \\mathcal{N} \\left(0,10\\times 10^6\\right)
+
+
+and we make the hypothesis that the observation errors are independent.
+We set the number of observations to:
+
+.. math::
+
+   n = 10.
+
+
+We generate a Monte-Carlo sample with size :math:`n`:
+
+.. math::
+
+   \\sigma_i = G(\\epsilon_i,R,C,\\gamma) + (\\epsilon_\\sigma)_i,
+
+
+for :math:`i = 1,..., n`.
+The observations are the pairs :math:`\\{(\\epsilon_i,\\sigma_i)\\}_{i=1,...,n}`,
+i.e. each observation is a couple made of the strain and the corresponding stress.
+
+Variables
+---------
+
+In the particular situation where we want to calibrate this model, the following
+list presents which variables are observed input variables, input calibrated
+variables and observed output variables.
+
+- :math:`\\epsilon`: Input. Observed.
+- :math:`R`, :math:`C`, :math:`\\gamma` : Inputs. Calibrated.
+- :math:`\\sigma`: Output. Observed.
+"""
+
+# %%
+import openturns as ot
+import openturns.viewer as viewer
+from matplotlib import pylab as plt
+from openturns.usecases import chaboche_model
+
+ot.Log.Show(ot.Log.NONE)
+
+# %%
+# Generate the observations
+# -------------------------
+# In practice, we generally use a data set which has been obtained from
+# measurements.
+# In this example, we generate the data using noisy observations of the
+# physical model.
+# In the next part, we will calibrate the parameters using the calibration
+# algorithms.
+
+# %%
+# Load the Chaboche model
+cm = chaboche_model.ChabocheModel()
+print("Inputs:", cm.model.getInputDescription())
+print("Outputs:", cm.model.getOutputDescription())
+
+# %%
+# We see that there are four inputs: `Strain`, `R`, `C` and `Gamma` and one
+# output: `Sigma`.
+# The `Strain` is observed on input and the stress `Sigma` is observed on
+# output.
+# Using these observations, we want to estimate the parameters `R`, `C`
+# and `Gamma`.
+
+# %%
+# We get the Chaboche model and the joint input distribution :
+g = cm.model
+inputDistribution = cm.inputDistribution
+print("inputDistribution:", inputDistribution)
+
+# %%
+# Set the calibration parameters
+# ------------------------------
+# In this part, we begin the calibration study.
```

Comment:
```suggestion
# Set the calibration parameters
# ------------------------------
#
# In this part, we begin the calibration study.
```
Otherwise the title is not recognized as such.

Reply:
Actually, it works well without that empty comment line, see:

![image](https://user-images.githubusercontent.com/31351465/232012646-716705e6-af73-4f26-b932-18e20f8036e2.png)

Reply:
Strange, I am pretty sure I saw a problem in the display. Perhaps it was something else and you have already fixed it.